.github/workflows/github-release.yml

```diff
@@ -0,0 +1,20 @@
+---
+name: GitHub Release
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+jobs:
+  release:
+    name: 'GitHub Release'
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Create GitHub release
+        uses: softprops/action-gh-release@v2
```

Comment:
we should use the official release workflow from github for this 🤔 .
for example see: https://github.com/voxpupuli/crafty/blob/main/.github/workflows/release.yml

Reply:
this can then use a config for the action like: https://github.com/voxpupuli/container-puppetserver/blob/main/.github/release.yml